doc/release-notes/5028-dataset-explore

```diff
@@ -0,0 +1,5 @@
+There is a DB migration at: 
+
+https://github.com/IQSS/dataverse/blob/5028-dataset-explore-btn/src/main/resources/db/migration/V4.15.1.1__5028-dataset-explore.sql
+
+We will need to reflect this in the release notes. Verify at release time. 
```

Comment:
Since this is a flyway script, and no interactive steps are needed for this migration - do we even need to mention this script in the release notes?

Reply:
I added this because I thought it was needed. I'll remove it.

Reply:
Removed in https://github.com/IQSS/dataverse/pull/6059/commits/c0df37dd564214fd53f3e05b722b97d3a453b0b6